examples/advanced/bionemo/downstream/sabdab/jobs/fedavg_sabdab_esm1nv/app/custom/base_config.yaml

```diff
@@ -4,13 +4,12 @@ do_testing: False # set to true to run evaluation on test data after training, r
 restore_from_path: null # used when starting from a .nemo file
 
 trainer:
-  devices: 2 # number of GPUs or CPUs
+  devices: 1 # number of GPUs or CPUs
   num_nodes: 1 
   accelerator: gpu #gpu or cpu
-  precision: 16 #16 or 32
+  precision: 16-mixed #16 or 32
```

Comment:
any restrictions if user want to use 16-mixed, are the variables inside this yml has documentation to explain what each parameter do?

Reply:
yes these are documented in the bionemo FW documentations and inside the bionemo FW yamls config examples.